package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "amazon-cognito-identity-js": "^5.2.9"
+  }
```

Comment:
Ah sorry I wasn't specific! I meant for you to run this under `website` only, not the top-level project. We want this to get added to `website/package.json`